File: problem99/problem99.rb
inputs = []
File.open('base_exp.txt').each_line do |line|
  inputs << line.chomp.split(',').map{|i| i.to_f}
end
puts "end reading file..."

max, line = 0, 0
inputs.each_with_index do |x, i|
  val = x[1]*Math.log10(x[0])
  max, line = val, i+1 if max < val
end

p inputs[line]
p line

File: problem99/problem99.py
from math import log

input
f = open('base_exp.txt', 'r')
input = [line.strip().split(',') for line in f]
input = [[float(x[0]), float(x[1])] for x in input]

max, line = input[0], 0
for i, x in enumerate(input):
	if max[1]*log(max[0]) < x[1]*log(x[0]):
		max, line = x, i

print max, line